### PR TITLE
Run tests on Travis with embedded API server

### LINKED ISSUE #2: Test calling the API from the UI
See https://github.com/c2corg/v6_ui/pull/1#issuecomment-145856898

### PR DESCRIPTION
**Pull request by tsauerwein**
When running the tests on Travis...

 * Clone the c2corg_api repo (master branch).
 * Install c2corg_api and set up a database.
 * Start an API server at localhost:5432.
 * Use this server to run the unit tests of the UI application.

By doing this we don't have to rely any more on an external API server.

Closes #2

### CODE REVIEW

**Comment by asaunier** [comment]
That looks great. Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -4,10 +4,16 @@ language: python
...
 
+addons:
+  postgresql: "9.4"
+
 install:
 - make -f config/travis .build/dev-requirements.timestamp
 - make -f config/travis install
 
+before_script:
+- scripts/travis.sh
```
Comment (elemoine) [inline .travis.yml]: Would it make sense to use a more specific script name?

**Comment by elemoine** [comment]
I just added a very minor comment. Looks great indeed.

**Comment by tsauerwein** [comment]
Thanks for the feedback!

**Pull request merged**